data/sgssb-data.txt

```diff
@@ -23,16 +23,18 @@ Signature Move: Mystic Mirage
 Notes: idk
 Rank: N/A.
 
-Gligars
-(Gligar)
-@ Eviolite
-EVs: 172 Atk / 84 Def / 252 SpD
+Lycanium Z
+(Lycanroc)
+@ Lycanium Z
```

Comment:
Isn't this going to be a real Z crystal in a few months? We shouldn't override the data of existing (or soon to do existing) moves/items/ect.

Reply:
Not the lycanium z i swear